internal/encoding/crypt3.go

```diff
@@ -0,0 +1,22 @@
+package encoding
+
+const encoding = "./0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+
```

Comment:
Lets give a bit more unique name for this constant.

```suggestion
const crypt3Encoding = "./0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
```